sdk/appcenter-identity/src/main/java/com/microsoft/appcenter/identity/Identity.java

```diff
@@ -402,22 +402,22 @@ public void run() {
 
                 @Override
                 public void onError(MsalException exception) {
-                    AppCenterLog.error(LOG_TAG, "User login failed.", exception);
+                    AppCenterLog.error(LOG_TAG, "User signIn failed.", exception);
                 }
 
                 @Override
                 public void onCancel() {
-                    AppCenterLog.warn(LOG_TAG, "User canceled login.");
+                    AppCenterLog.warn(LOG_TAG, "User canceled signIn.");
                 }
             });
         } else {
-            AppCenterLog.debug(LOG_TAG, "Login called while not configured or not in foreground, waiting.");
-            mLoginDelayed = true;
+            AppCenterLog.debug(LOG_TAG, "Sing In called while not configured or not in foreground, waiting.");
```

Comment:
nit: Typo

Reply:
We should also use the method name in logs

```suggestion
            AppCenterLog.debug(LOG_TAG, "signIn called while not configured or not in foreground, waiting.");
```